fleet/views.py

```
@@ -8685,42 +8700,58 @@ def multi_operator_mass_log_review(request):
+                    )
+                    result = _create_board_mass_log_trips(
+                        vehicle=vehicle,
+                        board_obj=board_obj,
+                        selected_date=selected_date,
+                        override_existing=bool(data.get('override')),
+                        clear_service_date=False,
+                    )
+                    total_created += result['created']
+                    total_skipped += result['skipped']
+                    total_overwritten += result['overwritten']
+                except Exception as exc:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)